src/app/developer/icreator/page.tsx

```
@@ -0,0 +1,153 @@
+import { SidebarProvider, SidebarTrigger } from '@/components/ui/sidebar'
+import AppSidebar from './components/app-sidebar'
+import { Input } from '@/components/ui/input'
+import { Label } from '@/components/ui/label'
+import { Switch } from '@/components/ui/switch'
+import { Button } from '@/components/ui/button'
+import InfoMenu from '@/components/originui/info-menu'
+import NotificationMenu from '@/components/originui/notification-menu'
+import { useRouter } from 'next/navigation'
+
+const ICreatorPage = () => {
+  const { } = useCreator()
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)